pkg/api/staking_test.go

```diff
@@ -26,7 +26,7 @@ func TestDepositStake(t *testing.T) {
 	t.Parallel()
 
 	txHash := common.HexToHash("0x1234")
-	minStake := big.NewInt(100000000000000000).String()
```

Comment:
I am just curious, why is minStake lowered?

Reply:
Very good catch. My mistake, minStake did not get lowered. Fixed it